sdk/rust-examples/audio-event-triangulation/src/main.rs

```diff
@@ -0,0 +1,138 @@
+//! An implementation of triangulation of audio event data
+//!
+//! This demo takes in audio events in the form of a timestamp,
+//! GPS coordinates, and an analog rich audio sample, calculates the
+//! best effort triangulated location, returning the estimated location
+//!
+//! ## Authors
+//!
+//! The Veracruz Development Team.
+//!
+//! ## Copyright
+//!
+//! See the file `LICENSING.markdown` in the Veracruz root directory for licensing
+//! and copyright information.
+
+use anyhow;
+use std::{
+    convert::TryInto,
+    fs,
+    io,
+};
+
+struct AudioEvent {
+    #[allow(dead_code)]
+    timestamp: u32,
+    location: (i32, i32),
+    samples: Vec<i16>,
+}
+
+impl AudioEvent {
+    fn power(&self) -> f64 {
```

Comment:
Can we have some `rustdoc` comments on the functions in this file, explaining what they are doing?